packages/editor/src/components/post-author/index.js

```diff
@@ -69,17 +85,21 @@ function PostAuthor() {
 			setFieldValue( postAuthor.name );
 		}
 	}, [ postAuthor ] );
-
 	/**
 	 * Handle author selection.
 	 *
-	 * @param {number} postAuthorId The selected Author.
+	 * @param {number|Object} selectedAuthor The selected Author.
 	 */
-	const handleSelect = ( postAuthorId ) => {
-		if ( ! postAuthorId ) {
+	const handleSelect = ( selectedAuthor ) => {
```

Comment:
Now handles selection from the combobox (the value only) or from the select element (the synthetic event)